modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.tsx

```diff
@@ -12,12 +12,75 @@
  * details.
  */
 
+import ClayButton from '@clayui/button';
+import ClayEmptyState from '@clayui/empty-state';
 import React from 'react';
 
 import BasePage from '../../components/BasePage';
+import {ESteps, TGenericComponent} from './WizardPage';
 
-const Step = () => {
-	return <BasePage title={Liferay.Language.get('property')}></BasePage>;
+interface IStepProps extends TGenericComponent {}
+
+const Step: React.FC<IStepProps> = ({onChangeStep}) => {
+	return (
+		<BasePage
+			description={Liferay.Language.get('property-description')}
+			title={Liferay.Language.get('property-assignment')}
+		>
+			<div className="text-right">
+				<ClayButton
+					displayType="secondary"
+
+					// TODO: Replace this alert with modal of LRAC-11980
+
+					onClick={() => alert('cria nova property')}
```

Comment:
Could you just return a empty fn?
```suggestion
					onClick={() => {}}
```

Reply:
@interaminense done!